patches/kilo/nova-pxe-boot.patch

```diff
@@ -1,14 +0,0 @@
-diff --git a/nova/virt/libvirt/driver.py b/nova/virt/libvirt/driver.py
```

Comment:
Probably this too, just so there's a patch available for reference.  I should check if this patch still applies cleanly though.